### PR TITLE
docs(v2): Add AttoBot website to showcase page

### PR DESCRIPTION
**Pull request by wolf20482**
## Motivation

The official website of AttoBot is using Docusaurus v2. It can be added on the showcase page.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/master/CONTRIBUTING.md#pull-requests)?

Yes.

## Test Plan

Run yarn run start:v2 and check that the page http://localhost:3000/showcase is rendered properly.

## Related PRs

N/A

### CODE REVIEW

File: website/src/data/users.js [hunk 1]
```diff
@@ -31,6 +31,15 @@
...
 const users = [
+  {
+    title: 'AttoBot',
+    description: 'A multi-purpose Discord bot with many features and API integrations that will enhance your Discord experience.',
+    preview: require('./showcase/attobot.png'),
+    website: 'https://attobot.xyz',
+    source: 'https://github.com/attobot-discord/website',
+    fbOpenSource: false,
+    pinned: true,
+  },
```
Comment (slorber) [inline website/src/data/users.js:42]: can you please use the alphabetical order?

**Review by slorber — CHANGES_REQUESTED** [review]
thanks 👍 

need to use alphabetical order + a better site screenshot uniform to the existing ones :)

File: website/src/data/users.js (hunk 1 above)
Comment (wolf20482) [reply website/src/data/users.js:42]: yes

Comment (wolf20482) [reply website/src/data/users.js:42]: wait

Comment (wolf20482) [reply website/src/data/users.js]: done

**Comment by slorber** [comment]
lgtm thanks

**Pull request merged**